frontend/packages/console-plugin-sdk/README.md

```diff
@@ -88,10 +88,10 @@ The plugin entry module is loaded immediately upon Console startup, so it should
   },
   "consolePlugin": {
     "entry": "src/plugin.tsx",
-    "integrationTests": {
+    "integrationTestSuites": {
```

Comment:
Nit: remove `devDependencies` above, internal packages should go away over time.